core/src/main/scala/kafka/server/DynamicConfig.scala

```diff
@@ -85,12 +85,10 @@ object DynamicConfig {
     def validate(props: Properties): util.Map[String, AnyRef] = DynamicConfig.validate(ipConfigs, props, customPropsAllowed = false)
 
     def isValidIpEntity(ip: String): Boolean = {
```

Comment:
this is used by `ConfigCommand` only, so could you please move this method to `ConfigCommand`?